config/prod/tthyer-arvados.yaml

```diff
@@ -0,0 +1,50 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-linux-v1.j2
+stack_name: tthyer-arvados-trial
+sceptre_user_data:
+  Distro: "ubuntu"     # (valid values: aws, ubuntu) AMIId must match distro
+  # (Optional) Expose ports to incoming traffic (default is no open ports) (valid range: 1-65535)
+  # OpenPorts: ["80","443"]
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "SysBio"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "amp-ad/arvados-trial"
+  # The resource owner
+  OwnerEmail: "tess.thyer@sagebase.org"
+
+  # Default account settings, leave alone unless you plan to override
+  VpcName: "sandcastlevpc"
+  KeyName: "sandbox"
+  AMIId: "{{stack_group_config.default_ubuntu_ami}}"
```

Comment:
This no longer works. Must add in AMI Id

Reply:
@thomasyu888 I'm doing this exact thing in toil-cluster-infra. Why doesn't it work here? Do you mean it is undesirable rather than not functional?

Reply:
I'm not sure actually... I defer to @zaro0508

Reply:
we no longer use the default from config.yaml, it's difficult to maintain.  please use the explicit AMI from the example template.